packages/console/src/pages/SignInExperience/components/TermsForm.tsx

```diff
@@ -0,0 +1,37 @@
+import { SignInExperience } from '@logto/schemas';
+import React from 'react';
+import { UseFormRegister, UseFormWatch } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import FormField from '@/components/FormField';
+import Switch from '@/components/Switch';
+import TextInput from '@/components/TextInput';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  register: UseFormRegister<SignInExperience>;
+  watch: UseFormWatch<SignInExperience>;
```

Comment:
Use `useFormContext` and also try `getValues` instead of `watch`

Reply:
updated